Assets/Scripts/Drone/DroneOnDewPickup.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DroneOnDewPickup : MonoBehaviour
+{
+    public static DroneOnDewPickup instance;
+
+    /// <summary>
+    /// Makes the drone rotate in a around the y axis and display the island heart when jellyDew is picked up the first time
+    /// </summary>
+
+    public bool didtheThing { get; private set; } = false;
+    [SerializeField] Sprite islandHeartImage;
+    [SerializeField] float rotationSpeed;
+    bool currentlyRotating = false;
+    float counter = 0;
+
+    void Start()
+    {
+        instance = this;
+    }
+
+    void Update()
+    {
+        if (currentlyRotating)
+        {
+            transform.Rotate(0, -rotationSpeed * 0.5f * Time.deltaTime, 0);
+            counter += Time.deltaTime * rotationSpeed;
+            Debug.LogWarning(counter);
+            if (counter > 2)
+            {
+                currentlyRotating = false;
+                ShowNewlyScannedThing.ShowImage(islandHeartImage);
+                didtheThing = true;
+            }
+        }
+    }
+
+    public void Activate()
+    {
+        if (!currentlyRotating)
+        {
+            currentlyRotating = true;
+            transform.rotation = Quaternion.identity;
```

Comment:
It probably shouldn't change the rotation here, because it'll snap to no rotation (breaking immersion). Instead, to remove any rotation which isn't around the vertical axis, it should smoothly rotate to remove that rotation.

Maybe the drone needs a target look direction in the movement code, and rotate towards that. I guess we can keep this snapping for now if you're not sure what to do.